packages/js/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "lib": ["ESNext"],
+    "target": "ESNext",
+    "module": "ESNext",
+    "strict": true,
+    "skipLibCheck": true,
+    "typeRoots": ["./node_modules/@types"],
+    "sourceMap": true,
+    "declaration": false,
```

Comment:
```suggestion
    "declaration": true,
```

Reply:
Types need to be generated for DX.

Reply:
the types are generated by tsup ;) but this option is misleading